Makefile

```diff
@@ -3,6 +3,10 @@ VENV_DIR = .venv
 IMAGE_NAME = themis
 VENV_RUN = . $(VENV_DIR)/bin/activate && export PYTHONPATH=.venv/lib64/python2.7/dist-packages
 AWS_PROFILE = faster
+POSTGRES_PORT_BIND=-p 5432:5432
+THEMIS_PORTBIND=-p 8080:8080
+THEMIS_DB_URL=postgresql://faster:faster_password@localhost:5432/faster
+
```

Comment:
remove empty line

Reply:
I think it would be nice a line between the variables and the first script